packages/grafana-data/src/field/fieldOverrides.test.ts

```diff
@@ -645,7 +645,7 @@ describe('getLinksSupplier', () => {
     expect(links[0]).toEqual(
       expect.objectContaining({
         title: 'testDS',
-        href: `/explore?left=${encodeURIComponent('{"datasource":"testDS","queries":["12345"],"panelsState":{}}')}`,
+        href: `/explore?left=${encodeURIComponent('{"datasource":"1234","queries":["12345"],"panelsState":{}}')}`,
```

Comment:
**Nit:** maybe we could extract the `datasourceUid` to a separate variable to remove some duplication
```javascript
const datasourceUid = "1234";
const f0 = new MutableDataFrame({
// ...
```